libs/portals/admin/form-system/src/screens/Form/Form.loader.ts

```diff
@@ -1,73 +1,40 @@
 import type { WrappedLoaderFn } from '@island.is/portals/core'
 import { FormSystemFormResponse } from '@island.is/api/schema'
-
-import { ApolloClient, NormalizedCacheObject } from '@apollo/client'
-import {
-  FormSystemGetFormDocument,
-  FormSystemGetFormQuery,
-} from './Form.generated'
-import {
-  FormSystemGetInputQuery,
-  FormSystemGetInputDocument,
-} from './GetInput.generated'
+import { GET_FORM } from '@island.is/form-system/graphql'
+import { removeTypename } from '../../lib/utils/removeTypename'
 
 export interface FormLoaderResponse {
   formBuilder: FormSystemFormResponse
-  client: ApolloClient<NormalizedCacheObject>
+}
+
+interface FormLoaderQueryResponse {
+  formSystemGetForm: FormSystemFormResponse
```

Comment:
This does not provide the same level of type safety, as the query may include a subset of the fields in the schema. Can use codegen in `@island.is/form-system/graphql` and re-export anything useful.